encodings/bytebool/src/array.rs

```diff
@@ -90,6 +92,14 @@ impl ArrayVariants for ByteBoolArray {
 }
 
 impl BoolArrayTrait for ByteBoolArray {
+    fn invert(&self) -> VortexResult<Array> {
+        ByteBoolArray::try_from_vec(
+            self.maybe_null_slice().iter().map(|v| !v).collect(),
```

Comment:
what is the definition of a true value in this encoding, `1` or `0xFF` ? any non-zero value?

Reply:
if it's 1 this should be `1 - v`. but we should probably document this explicitly somewhere

Reply:
on disk it will be `1` but in this case this is a in memory value that's been transmuted to a valid `bool`